frontend/src/components/home.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import Image from 'next/image';
+import BitcoinLogo from '../assests/svgs/bitcoin-logo.svg';
+import Machines from '../assests/images/machines.webp';
+import styles from './styles.module.css';
+
+const HomePage = () => {
+    return (
+        <div
+            className={`flex w-full max-w-full min-h-screen overflow-scroll md:grid md:gap-0 md:h-screen md:max-h-screen md:overflow-hidden ${styles.wrapper}`}
+        >
+            <div className='bg-secondary-transparent-black flex gap-7 flex-col p-5 py-12 md:p-8 xl:p-12 md:py-12 max-w-full md:max-w-xl w-full md:bg-primary-transparent-black z-50'>
+                <div className='flex flex-col gap-6'>
+                    <section className='flex gap-3 items-center justify-items-center'>
+                        <Image src={BitcoinLogo} alt='bitcoin logo' className='w-12 h-12 md:w-12 md:h-12 xl:w-20 xl:h-20' />
+                        <h2 className='font-bold xl:text-custom-logo text-5xl md:text-5xl xl:text-6xl leading-none text-custom-purple'>warnet</h2>
+                    </section>
+                    <p className='text-2xl md:text-2xl lg:text-3xl xl:text-4xl'>Monitor and analyze the emergent behaviors of P2P networks</p>
+                </div>
+
+                <div className='flex flex-col gap-9 text-base md:text-lg lg:text-2xl font-normal text-custom-gray'>
+                    <p>Build a stronger, more resilient Bitcoin</p>
+                    <p>Break the warnet before it comes to mainnet</p>
+                    <p>Monitor extreme and yet unknown network behaviors</p>
+                    <a
```

Comment:
lets make this open in another tab
```suggestion
                    <a target="_blank"
```